client/src/components/router/RoutePaths.ts

```diff
@@ -0,0 +1,13 @@
+type RoutePaths = {
```

Comment:
Moved this into its own file as it was causing a circular dependency in `LandingPage`.